python/cuml/test/test_nearest_neighbors.py

```diff
@@ -42,6 +42,10 @@ def predict(neigh_ind, _y, n_neighbors):
     import scipy.stats as stats
 
     neigh_ind = neigh_ind.astype(np.int32)
+    if isinstance(_y, cp.core.core.ndarray):
```

Comment:
It'd be nice if we had a very simple wrapper somewhere to do this "y = ensure_is_numpy(y)" - maybe we already do? (Certainly we have the input_utils functions meant for conversion on function entry, though they may be overkill in complexity) We can discuss a bit offline